bundles/binding/org.openhab.binding.zwave/database/schlage/be468.xml

```diff
@@ -0,0 +1,109 @@
+        <Class><id>0x71</id></Class>
+        <Class><id>0x72</id></Class>
+        <Class><id>0x80</id></Class>
+        <Class><id>0x84</id></Class>
+        <Class><id>0x85</id></Class>
+        <Class><id>0x86</id></Class>
+        <Class><id>0x98</id></Class>
+    </CommandClasses>
+
+    <Configuration>
+        <Parameter>
+            <Index>3</Index>
+            <Type>list</Type>
+            <Default>255</Default>
+            <Size>1</Size>
+            <Label lang="en">Beeper</Label>
+            <Help lang="en">Audio feedback when keypad pressed during normal operation</Help>
+            <Item>
+                <Value>0</Value>
+                <Label lang="en">Disable</Label>
+            </Item>
+            <Item>
+                <Value>255</Value>
+                <Label lang="en">Enable</Label>
+            </Item>
+        </Parameter>
+        <Parameter>
+            <Index>4</Index>
+            <Type>list</Type>
+            <Default>0</Default>
+            <Size>1</Size>
+            <Label lang="en">Vacation Mode</Label>
+            <Help lang="en">Prevents all user codes from unlocking the deadbolt</Help>
+            <Item>
+                <Value>0</Value>
+                <Label lang="en">Disable</Label>
+            </Item>
+            <Item>
+                <Value>255</Value>
+                <Label lang="en">Enable</Label>
+            </Item>
+        </Parameter>
+        <Parameter>
+            <Index>5</Index>
+            <Type>list</Type>
+            <Default>255</Default>
+            <Size>1</Size>
+            <Label lang="en">Lock &amp; Leave</Label>
+            <Help lang="en">Press the Schlage button to lock the deadbolt</Help>
+            <Item>
+                <Value>0</Value>
+                <Label lang="en">Disable</Label>
+            </Item>
+            <Item>
+                <Value>255</Value>
+                <Label lang="en">Enable</Label>
+            </Item>
+        </Parameter>
+        <Parameter>
+            <Index>15</Index>
+            <Type>list</Type>
+            <Default>0</Default>
+            <Size>1</Size>
+            <Label lang="en">Auto Lock</Label>
+            <Help lange="en">Automatically locks 30 seconds after unlocking</Help>
+            <Item>
+                <Value>0</Value>
+                <Label lang="en">Disable</Label>
+            </Item>
+            <Item>
+                <Value>255</Value>
+                <Label lang="en">Enable</Label>
+            </Item>
+        </Parameter>
+        <Parameter>
+            <Index>16</Index>
+            <Type>byte</Type>
+            <Default>4</Default>
+            <Size>1</Size>
+            <Minimum>4</Minimum>
+            <Maximum>8</Maximum>
+            <Label lange="en">User Code PIN Length</Label>
+            <Help lang="en">User codes can be 4-8 digits. Changing length will delete all existing codes.</Help>
+        </Parameter>
+    </Configuration>
+
+    <Associations>
```

Comment:
I don't know what the association groups do, but if one (or both) of them are meant to report information to a controller then you should set ```<SetToController>true</SetToController>``` in the group.